src/main/cpp/blaze_util_windows.cc

```diff
@@ -700,6 +702,13 @@ int ExecuteDaemon(const blaze_util::Path& exe,
   CmdLine cmdline;
   CreateCommandLine(&cmdline, exe, wesc_args_vector);
 
+  std::wstringstream newcmdline;
+  newcmdline 
+    << L"C:\\Program Files (x86)\\IncrediBuild\\IbConsole.exe "
+    << "/Title=\"Bazel Build\" /AllowRemote=\"cl,link\" /SILENT /Command=\""
+    << cmdline.cmdline << "\"";
+  wcsncpy(cmdline.cmdline, newcmdline.str().c_str(), MAX_CMDLINE_LENGTH - 1);
```

Comment:
Based on my understanding of the documentation and personal testing, it seems that IncrediBuild only distributes directly spawned child processes to remote machines. As a result, it may not be feasible to execute Bazel commands directly with IBConsole.

Reply:
It may be more reasonable to define a `java_command_line_format` parameter, and then use `--java_command_line_format=“IBConsole /Command="$"` to execute bazel.